src/RoslynPad.Common/Editor/CustomCompletionWindow.cs

```diff
@@ -0,0 +1,20 @@
+using System.Windows;
+using System.Windows.Input;
+using ICSharpCode.AvalonEdit.CodeCompletion;
+using ICSharpCode.AvalonEdit.Editing;
+
+namespace RoslynPad.Editor
+{
+    public class CustomCompletionWindow : CompletionWindow
```

Comment:
Couldn't you apply the style to the `CompletionWindow` class? I see you haven't changed the `DefaultStyleKey` property.